programming_examples/basic/shuffle_transpose/kernel.cc

```diff
@@ -0,0 +1,78 @@
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2024 AMD Inc.
```

Comment:
```suggestion
// (c) Copyright 2025 AMD Inc.
```

This should be moved to `aie_kernels/aie2`

Reply:
Please rename the file appropriately